go.mod

```diff
@@ -42,34 +42,33 @@ require (
 	github.com/initia-labs/OPinit v0.4.0
 	// we also need to update `LIBMOVEVM_VERSION` of Dockerfile#5
 	github.com/initia-labs/movevm v0.3.4
-	github.com/noble-assets/forwarding/v2 v2.0.0-20240514101621-172acc02aac6
+	github.com/noble-assets/forwarding/v2 v2.0.0-20240521090705-86712c4c9e43
 	github.com/pelletier/go-toml v1.9.5
 	github.com/pkg/errors v0.9.1
 	github.com/rakyll/statik v0.1.7
 	github.com/skip-mev/block-sdk/v2 v2.1.2
-	github.com/skip-mev/slinky v0.4.3
+	github.com/skip-mev/slinky v1.0.4
```

Comment:
Recommend bumping to 1.0.5. We fixed https://github.com/skip-mev/slinky/pull/624 in that version which will probably cause issues when people upgrade to 1.0.4.